packages/cli/templates/default-plugin/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint.js')],
```

Comment:
this ESLint config is required to make the glob paths for things like the `tsx`-specific overrides work properly within the plugins.